engine/src/main/java/org/terasology/particles/components/BlockBreakEffectComponent.java

```diff
@@ -13,7 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.terasology.particles.components;
 
-@API package org.terasology.logic.particles;
+import org.terasology.math.geom.Vector2f;
+import org.terasology.network.Replicate;
 
-import org.terasology.module.sandbox.API;
+/**
+ *
+ */
+public class BlockBreakEffectComponent extends ParticleSystemComponent {
+
+    @Replicate
+    public boolean randBlockTexDisplacement;
+
+    @Replicate
+    public Vector2f randBlockTexDisplacementScale = new Vector2f(0.25f, 0.25f);
+
+    public BlockBreakEffectComponent() {
```

Comment:
An explictly defined constructor is a bit strange in a component, I would just assign a value to the field like for the Vector.